README.md

```diff
@@ -1 +1,58 @@
-# Space-Travelers-Hub
\ No newline at end of file
+# Space-Travelers-Hub
+
+![screenshot](./SC1.png)
+
+![screenshot](./SC2.png)
+
+![screenshot](./SC3.png)
+
+Space-Travelers-Hub is an webapp where you can reserve a rocket and join a mission in outer space. Enjoy the interstellar voyage. Mars here we come!
+
+## Live demo
```

Comment:
- [ ] Kindly update the live demo in your README to reflect the app's current state. Even though the rockets page is displayed by default locally, you need to change this during deployment.